metasrv/src/api/flight_server.rs

```diff
@@ -92,24 +96,50 @@ impl FlightServer {
             self.conf
         );
 
-        let meta_config = &self.conf.raft_config;
+        let raft_config = &self.conf.raft_config;
 
-        let mn = if meta_config.boot {
-            MetaNode::boot(0, meta_config).await?
-        } else if meta_config.single {
+        let mn = if raft_config.boot {
+            MetaNode::boot(0, raft_config).await?
+        } else if raft_config.single {
             let (mn, _is_open) =
-                MetaNode::open_create_boot(meta_config, Some(()), Some(()), Some(())).await?;
+                MetaNode::open_create_boot(raft_config, Some(()), Some(()), Some(())).await?;
```

Comment:
If we should check `_is_open` is true to ensure metanode is available?

Reply:
`_is_open` indicates whether it is opened from a previous state, not created.

If it does not return an error, meta node is definitely available.

Reply:
Got it.